graphistry/dep_manager.py

```diff
@@ -0,0 +1,30 @@
+import importlib
+
```

Comment:
this class benefits from a comment on its design and the problem it solves

Reply:
ex: maybe it does a fast & cached check of a package being in the path before actually importing, so there should be a testable and observable speedup?